packages/composite-checkout/src/components/radio-button.tsx

```diff
@@ -1,4 +1,5 @@
 import styled from '@emotion/styled';
+import clsx from 'clsx';
```

Comment:
The `composite-checkout` package does not have `clsx` as a dependency. This might work due to the nature of the monorepo but we shouldn't rely on it as it could break at any time. Do you need `clsx()` here? Or could you do this manually?

Reply:
Good catch. Done.